ast/ast.go

```diff
@@ -2051,6 +2053,29 @@ type DropTable struct {
 	Name     *Ident
 }
 
+// RenameTable is RENAME TABLE statement node.
+//
+//	RENAME TABLE {{.SrcDests | sqlJoin ", "}}
+type RenameTable struct {
+	// pos = Drop
+	// end = Name.end
+
+	Rename token.Pos // position of "RENAME" pseudo keyword
+
+	SrcDests []*RenameTableSrcDest // len(SrcDests) > 0
+}
+
+// RenameTableSrcDest is old_table_name TO new_table_name node in RENAME TABLE statement.
+//
+//	{{.Src | sql}} TO {{.Dest | sql}}
+type RenameTableSrcDest struct {
```

Comment:
`RenameTableTo` sounds like a better name to me than this. At least, it seems that "src" and "dest" do not appear in the document.

Reply:
It is reasonable comment and LTGM `RenameTableTo `.
Fixed.